src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.io.File;
```

Comment:
I like how you have organised your imports according to their functionality. It's clear and easy to read. Great job!